pkg/estimator/client/cache.go

```diff
@@ -79,11 +79,11 @@ func (c *SchedulerEstimatorCache) GetClient(name string) (estimatorservice.Estim
 }
 
 // EstablishConnection establishes a new gRPC connection with the specified cluster scheduler estimator.
-func EstablishConnection(name string, estimatorCache *SchedulerEstimatorCache, port int) error {
+func EstablishConnection(name, namespace string, estimatorCache *SchedulerEstimatorCache, port int) error {
 	if estimatorCache.IsEstimatorExist(name) {
 		return nil
 	}
-	serverAddr := fmt.Sprintf("%s:%d", names.GenerateEstimatorServiceName(name), port)
+	serverAddr := fmt.Sprintf("%s.%s.svc.cluster.local:%d", names.GenerateEstimatorServiceName(name), namespace, port)
```

Comment:
It is better not to be fixed as `svc.cluster.local`, refering #2338

Reply:
thanks for your review , i know what you mean